### ISSUE TITLE
Replace full lodash import with import of used methods

### ISSUE THREAD

**Issue by Snowshield** [mkay581/tooltip-js#4]
Src code weight of your library - 5kb
Dist code weight - 90kb

If you replace full lodash import with import of used methods you can easily reduce weight of your library 10 times

**Comment by mkay581** [comment]
You are absolutely right. I haven't had much time to dedicate to this lib. You mind submitting a PR?
